src/gc/gc.cpp

```diff
@@ -18786,13 +18786,13 @@ void gc_heap::fix_card_table ()
                     time_stop - time_start, tot_cycles);
 #endif //TIME_WRITE_WATCH
 
-            assert( ((card_size * card_word_width)&(OS_PAGE_SIZE-1))==0 );
```

Comment:
On Windows, this really is the true operating system page size and not an arbitrary constant. The array of pointers that we receive from `GetWriteWatch` (used to track writes on the heap during concurrent GCs) are page-sized. When we're looping over this array, we're looking at single OS pages and setting cards for it.

I don't think that `GetWriteWatch` supports anything other than 4k pages, so it's probably fine to replace these uses with 4096, but perhaps we should assert somewhere that if we are using `GetWriteWatch`, `OS_PAGE_SIZE` is what we expect.